docs/create-packages/creating-a-package-msbuild.md

```diff
@@ -86,7 +86,7 @@ If you are using MSBuild with a non-SDK-style project and PackageReference, add
    ```xml
    <ItemGroup>
      <!-- ... -->
-     <PackageReference Include="NuGet.Build.Tasks.Pack" Version="5.2.0"/>
+     <PackageReference Include="NuGet.Build.Tasks.Pack" Version="6.7.0" PrivateAssets="all" Pack="false" />
```

Comment:
In another 3 years, this package version is going to be very old and won't support features that the current NuGet at the time will support.  I know people love examples that can be copy-pasted. I like deterministic tools, so I don't like floating versions as a feature, so I don't recommend `Version="*"`. Are there any other alternatives?

`Version="get latest version from nuget.org"`? `Version="$(LatestNuGetVersion)"`?  Links don't work in code blocks, so before or after the code block, we could have a link to https://www.nuget.org/packages/NuGet.Build.Tasks.Pack#versions-body-tab

There's no way we're going to remember to update this doc every release. Even if we do, people will get frustrated when pack hasn't had any meaningful updates, but we have to update this version anyway. 

I can't think of any obvious best solution.

Reply:
The docs can reference "code blocks", like here: https://learn.microsoft.com/en-us/nuget/reference/nuget-client-sdk#list-package-versions 

Can they reference csproj blocks too?

Reply:
Added a note here as suggested by @JonDouglas